test/integration/action_execution.go

```diff
@@ -255,6 +255,48 @@ var _ = Describe("Action Executor ", func() {
 		})
 	})
 
+	// Horizontal scale test
+	Describe("Executing horizontal scale action on a deployment", func() {
+		It("should match the expected replica number", func() {
+			// create a deployment with 2 replicas
+			dep, err := createDeployResource(kubeClient, depSingleContainerWithResources(namespace, "", 2, false, false, false))
+			framework.ExpectNoError(err, "Error creating test resources")
+
+			pod, err := getDeploymentsPod(kubeClient, dep.Name, namespace, "")
+			framework.ExpectNoError(err, "Error getting deployments pod")
+			// This should not happen. We should ideally get a pod.
+			if pod == nil {
+				framework.Failf("Failed to find a pod for deployment: %s", dep.Name)
+			}
+
+			targetNodeName := getTargetSENodeName(f, pod)
```

Comment:
Why do we need this here? Copy paste overload?

Reply:
will delete it